sql/geometry.go

```diff
@@ -34,9 +34,14 @@ type GeometryType struct {
 }
 
 // GeometryValue is the value type returned from GeometryType, which is an interface over the following types:
-// Point, Polygon, LineString.
+// Point, Polygon, LineString, MultiPoint, MultiLineString, MultiPolygon, GeometryCollection.
 type GeometryValue interface {
 	implementsGeometryValue()
+	GetSRID() uint32
+	SetSRID(srid uint32) GeometryValue
+	Serialize() []byte
+	WriteData(buf []byte)
+	Swap() GeometryValue
```

Comment:
add doc comments here. some of these methods are unused, I assume more changes are to come